pwiz_tools/Skyline/Model/TransitionGroupDocNode.cs

```diff
@@ -50,14 +50,16 @@ public class TransitionGroupDocNode : DocNodeParent
         /// </summary>
         /// <param name="id">The <see cref="TransitionGroup"/> identity for this node</param>
         /// <param name="children">A set of explicit children, or null if children should be auto-managed</param>
-        /// <param name="explicitTransitionGroupValues">Optional values like ion mobility etc</param>
-        public TransitionGroupDocNode(TransitionGroup id, TransitionDocNode[] children, ExplicitTransitionGroupValues explicitTransitionGroupValues = null)
+        /// <param name="explicitPrecursorFilter">Optional values like explicit ion mobility, CE etc</param>
+        /// <param name="libraryPrecursorFilter">Library (imsdb or blib) ion mobility value, if known</param>
+        public TransitionGroupDocNode(TransitionGroup id, TransitionDocNode[] children, PrecursorFilter explicitPrecursorFilter = null, PrecursorFilter libraryPrecursorFilter = null)
             : this(id,
                    Annotations.EMPTY,
                    null,
                    null,
                    null,
-                   explicitTransitionGroupValues ?? ExplicitTransitionGroupValues.EMPTY,
+                   explicitPrecursorFilter,

```

Comment:
There should probably be a class which has two properties: "ExplIcitPrecursorFilter" and "LibraryPrecursorFilter" and that one object is what gets passed to this constructor. Anything that we can do to reduce the number of parameters that this constructor takes is a good thing. I also have a feeling that there might be other things that we would want to add to this new class when it exists.